utils.py

```diff
@@ -20,23 +17,19 @@
 import matplotlib.image as mpimg
 
 
-
-def take_screenshot():
-    screen = wx.ScreenDC()
-    bmp = wx.Bitmap(Screenshot.SRC_W, Screenshot.SRC_H)
-    mem = wx.MemoryDC(bmp)
-    mem.Blit(0, 0, Screenshot.SRC_W, Screenshot.SRC_H, screen, Screenshot.OFFSET_X, Screenshot.OFFSET_Y)
-    return bmp
-
-
-
 def prepare_image(img):
+    import wx
```

Comment:
Awesome, I didn't spot that, but I think you're right. I'll give it a shot and update it if it works.